docs/charts.md

```diff
@@ -49,20 +49,22 @@ kubeVersion: A SemVer range of compatible Kubernetes versions (optional)
 description: A single-sentence description of this project (optional)
 keywords:
   - A list of keywords about this project (optional)
-home: The URL of this project's home page (optional)
+home: The URL of this project's home page (recommended)
 sources:
   - A list of URLs to source code for this project (optional)
 maintainers: # (optional)
   - name: The maintainer's name (required for each maintainer)
     email: The maintainer's email (optional for each maintainer)
     url: A URL for the maintainer (optional for each maintainer)
 engine: gotpl # The name of the template engine (optional, defaults to gotpl)
-icon: A URL to an SVG or PNG image to be used as an icon (optional).
-appVersion: The version of the app that this contains (optional). This needn't be SemVer.
+icon: A URL to an SVG or PNG image to be used as an icon (recommended).
+appVersion: The version of the app that this contains (recommended). This needn't be SemVer.
 deprecated: Whether this chart is deprecated (optional, boolean)
 tillerVersion: The version of Tiller that this chart requires. This should be expressed as a SemVer range: ">2.0.0" (optional)
 ```
 
+Recommended fields if missing will fail linting.
```

Comment:
The current linter doesn't fail on the recommended ones. Is this a request to change that behavior? For example, the linter doesn't check for `appVersion`. The code that checks a chartfile is [here](https://github.com/kubernetes/helm/blob/58e4b6ac61938fe056037826225dd19562c0ed9f/pkg/lint/rules/chartfile.go#L35)

Reply:
The linter being used by https://github.com/kubernetes/charts does fail when these things are missing.  I have had to add them to charts to get the CI to pass PRs I have submitted.

Reply:
Here is a PR that failed until I added `appVersion`
https://circleci.com/gh/kubernetes/charts/6777